src/aux-server/aux-web/shared/interaction/BaseInteractionManager.ts

```diff
@@ -614,7 +614,7 @@ export abstract class BaseInteractionManager {
             file.contextGroup.simulation3D.simulation.filePanel
         );
 
-        file.contextGroup.simulation3D.simulation.recent.addFileDiff(file.file);
+        //file.contextGroup.simulation3D.simulation.recent.addFileDiff(file.file);
```

Comment:
Are we going to use this line again? Maybe it should be removed instead of commented out?

Reply:
probably not as this was used to help set up the mod set on file deselect which we don't want anymore I can remove it.

Reply:
changed